### ISSUE TITLE
Accessibility: Tabs in are are missing focus state

### ISSUE THREAD

**Issue by oanamangiurea** [grafana/grafana#33849]
The new tab style is missing the focus state. They should show a blue border around as buttons and input have. 
 

![Screenshot 2021-05-10 at 15 27 26](https://user-images.githubusercontent.com/5183519/117659125-3bbe1d00-b1a4-11eb-93f1-09337bccacfb.png)

**Comment by sidverma32** [comment]
Hi @torkelo , I would like to pick this up. If you can guide me how to proceed from here?

**Comment by torkelo** [comment]
Something like this:

--- a/packages/grafana-ui/src/components/Tabs/Tab.tsx
+++ b/packages/grafana-ui/src/components/Tabs/Tab.tsx
@@ -7,6 +7,7 @@ import { Icon } from '../Icon/Icon';
 import { IconName } from '../../types';
 import { stylesFactory, useTheme2 } from '../../themes';
 import { Counter } from './Counter';
+import { getFocusStyles } from '../../themes/mixins';

 export interface TabProps extends HTMLProps<HTMLAnchorElement> {
   label: string;
@@ -62,15 +63,15 @@ const getTabStyles = stylesFactory((theme: GrafanaTheme2) => {
     link: css`
       color: ${theme.colors.text.secondary};
       padding: ${theme.spacing(1.5, 2, 1)};
+      display: block;
+      height: 100%;

       svg {
         margin-right: ${theme.spacing(1)};
       }

-      a {
-        display: block;
-        height: 100%;
-        color: ${theme.colors.text.secondary};
+      &:focus-visible {
+        ${getFocusStyles(theme)}
       }
     `,
     notActive: css`

**Comment by sidverma32** [comment]
Thanks for guiding me. I have raised the PR #34422 , please review and let me know if any modification is required.

**Comment by torkelo** [comment]
@oanamangiurea this is now fixed :)
